src/coreclr/vm/ceeload.cpp

```diff
@@ -414,6 +414,8 @@ void Module::Initialize(AllocMemTracker *pamTracker, LPCWSTR szName)
 
     m_loaderAllocator = GetAssembly()->GetLoaderAllocator();
     m_pSimpleName = m_pPEAssembly->GetSimpleName();
+    m_path = m_pPEAssembly->GetPath();
```

Comment:
```suggestion
    m_path = m_pPEAssembly->GetPath().GetUnicode();
```

I realize there's a `operator const WCHAR * () const` but I find implicit conversion operators hard to think about.